Tests/webcomponents/mixins/ez-sandbox-yui-component.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+        <title>Sandbox YUI component mixin test</title>
+
+        <script src="../../../webcomponentsjs/webcomponents-lite.js"></script>
+        <script src="../../../web-component-tester/browser.js"></script>
+
+        <link rel="import" href="/Resources/public/webcomponents/mixins/ez-sandbox-yui-component.html">
+        <script>
+        window.addEventListener('WebComponentsReady', function () {
```

Comment:
all that is completely unneeded as well. Removing `window.customElements.define` call means also removing that